packages/serverless-deploy-iam/bin/app.ts

```diff
@@ -443,6 +443,14 @@ export class ServiceDeployIAM extends cdk.Stack {
       type: new Group(this, `${serviceName}-deployers`),
       policies: [
         dummyPolicy,
+        {
+          name: "IAM",
```

Comment:
We already have one named `IAM` so this will cause issues. You can probably just called it `SERVICE_LINKED_ROLE`